processor/src/trace/utils.rs

```diff
@@ -237,6 +237,118 @@ impl HintCycle for u64 {
     }
 }
 
+// TRACE INFO
+// ================================================================================================
+
+/// Contains the data about lengths of the trace parts.
+///
+/// - `main_trace_len` contains the length of the main trace.
+/// - `range_table_len` contains the length of the range checker table.
+/// - `chiplet_lenghts` contains the trace lenghts of the all chiplets (hasher, bitwise, memory,
+/// kernel ROM)
+/// - `max_len` contains the trace length required to hold all execution trace steps (maximum
+/// length among range table length, main trace length and chiplets trace length).
+/// - `padded_len` contains `max_len` padded to the next power of two.
+#[allow(dead_code)]
```

Comment:
Why do we need `#[allow(dead_code)]` here? Shouldn't this be a part of a public interface?